integration-tests/jcache/src/test/java/org/apache/camel/quarkus/component/jcache/it/JcacheTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.jcache.it;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+class JcacheTest {
+
+    @Test
+    public void loadComponentJcache() {
+        /* A simple autogenerated test */
+        RestAssured.get("/jcache/load/component/jcache")
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
+    public void testCache() {
+        RestAssured.get("/jcache/sayHello")
+                .then()
+                .statusCode(200)
+                .body(is("Hello World : Non-Cached Response"));
+
+        RestAssured.get("/jcache/sayHello")
+                .then()
+                .statusCode(200)
+                .body(is("Hello World : Cached Response"));
+
+        //Wait till Cache expires
+        try {
+            Thread.sleep(12000);
```

Comment:
How do we know it is 12s ? It's a like a cache default value of 10s + 2s security ?

Reply:
I'd vote for a shorter cache expiration, if possible. 12s sounds too much.

Reply:
OK, I changed it to 3 seconds cache expiration and 4 seconds sleep, is it ok?

Reply:
LGTM

Reply:
> OK, I changed it to 3 seconds cache expiration and 4 seconds sleep, is it ok?

Much better, thanks!